gateway-js/src/index.ts

```diff
@@ -111,6 +112,9 @@ export type Experimental_DidResolveQueryPlanCallback = ({
 }: {
   readonly queryPlan: QueryPlan;
   readonly serviceMap: ServiceMap;
+  // Note that the `queryPlanPointer` on the operation context is *not guarenteed
+  // to be valid*. It may be cleaned up at any point, you cannot assume that it
+  // can be passed to query-planner-wasm without error.
```

Comment:
Is it right to say that the pointer is guaranteed to be valid while the operation that the operationContext was created for is being executed? 

Or is there a case where we might release the pointer while `_executor` is being run?

Reply:
That's right, the pointer will remain valid for the duration for `_executor`. I'll add a note about that.